docs/commerce/payment/configure_payment.md

```diff
@@ -0,0 +1,47 @@
+---
+description: Configure the payment process.
+edition: commerce
+---
+
+# Configure payment
+
+When you work with your Commerce implementation, you can review and modify the payment configuration.
+
+!!! note "Permissions" 
+
+    When you modify the workflow configuration, make sure you properly set user [permissions](permission_use_cases.md#commerce) for the Payment component.
+
+## Payment workflow
+
+Payment workflow relies on a [Symfony Workflow](http://symfony.com/doc/5.4/components/workflow.html).
+Each transition represents a separate payment step. 
+
+The default fallback workflow is `ibexa_payment`.
+
+### Default payment workflow configuration
+
+The default payment workflow configuration is called `ibexa_payment`, you can replace it with your custom workflow identifier if needed.
+
+To see the default workflow, in your project directory, navigate to the following file: `vendor/Ibexa/payment/src/bundle/Resources/config/workflow.yaml`.
+
+### Custom payment workflows
```

Comment:
It's important to add here configuration options which are available as part of the place/transition metadata e.g. label, translation_domain, colors etc.